### ISSUE TITLE
Goals of this project?

### ISSUE THREAD

**Issue by devinsmith** [stefan11111/gtk2#17]
This project look interesting but I wonder what the end goal is? I understand that stripping pieces out makes the project smaller and more maintainable. 

Also, sadly, many Linux distros seem to be removing older GTK packages so I could understand keeping a small version around for older software.

Finally, are there parts of the project you are looking for help with?

**Comment by stefan11111** [comment]
Hi.

The main purpose of this is for an alternative gtk2 with no accessibility support and no dependency on atk to exist.
I believe that accessibility support should not be mandatory, but optional, and this project achieves that(by removing it altogether).

Since gtk2 is EoL, only maintaining a patch, instead of a fork, wouldn't make much sense. Since I started a fork, I decided to also remove other features that are seldom used and that I feel needlessly waste cpu time.

I know that packages are moving towards gtk3 and gtk4. I use palemoon and leafpad, both of which can make use of gtk2. I have also tested this with steam, but I use the bundled gtk2 version with it.

This project should still be useful for other gtk2 apps.

I can't do much more with this repo, other than fix some bugs. If I remove a non-static function, I break the api. If I add a new function, It won't be used by other existing apps, unless this repo somehow becomes insanely popular. If you have any ideas as to what I should add to this project, feel free to tell me.

When gtk3 becomes EoL, I will probably do something similar with it too.
It's pointless to do this while gtk3 is still upstream supported, since I'd have to manually apply almost every upstream commit if I want it to be updated.
